doc/gui/examples/controls/collapse_data_view.py

```diff
@@ -0,0 +1,25 @@
+from taipy.gui import Gui
+import json
+
+# Sample JSON data for testing
+json_data = {
+    "name": "Sample Data",
+    "type": "Example JSON",
+    "attributes": {
+        "id": 123,
+        "description": "A simple JSON structure for demonstration",
+        "values": [1, 2, 3, 4, 5],
+        "nested": {
+            "flag": True,
+            "count": 10
+        }
+    }
+}
+
+# Define the page with the JSON viewer
+page = """
+<|{json_data}|json|expandable|>
```

Comment:
did you try to run this ?
This should be something like

```
import taipy as tp
import taipy.gui.builder as tgb
from taipy import Config, Orchestrator, Scope
from taipy.gui import Gui

json_config_node = Config.configure_json_data_node(
    id="json_node",
    default_path="./demo-1248-dn.json",
    scope=Scope.GLOBAL,
)

with tgb.Page() as data_node_viewer:
    tgb.html("h2", "Discovering Data Node json viewer")

    tgb.data_node(
        data_node="{json_data_node}"
    )
gui = Gui(page=data_node_viewer)

if __name__ == "__main__":
    orchestrator = Orchestrator()
    orchestrator.run()
    json_data_node = tp.create_global_data_node(json_config_node)
    gui.run(title="1248 json data_node")
```

Reply:
Yes sir , I did try running that. This is what it was showing
![image](https://github.com/user-attachments/assets/d8120353-40a4-4284-a5ae-e9b590e5bf44)